test/e2e_node/density_test.go

```diff
@@ -467,12 +467,12 @@ func getPodStartLatency(node string) (e2emetrics.KubeletLatencyMetrics, error) {
 
 	for _, samples := range ms {
 		for _, sample := range samples {
-			if sample.Metric["__name__"] == kubemetrics.KubeletSubsystem+"_"+kubemetrics.PodStartDurationKey {
```

Comment:
I don't have much context here, but I don't feel like this should be a part of the change.

Reply:
Fair enough; test file reverted